examples/oic.r.sacl-example2.json

```diff
@@ -29,35 +29,7 @@
           }
```

Comment:
I don't believe it was ever addressed whether amacl and sacl should be updated to refer to a list of ACE2. And on top of that, I believe the definitions need to be updated to contain an array of ACEs. Either way, this should be dealt with as a separate CR/Pull Request.